app/gateway/2.7.x/configure/auth/kong-manager/oidc-mapping.md

```diff
@@ -0,0 +1,172 @@
+---
+title: OIDC Authenticated Group Mapping
+badge: enterprise
+---
+
+Using Kong's OpenID Connect plugin (OIDC), you can map identity provider (IdP)
+groups to Kong roles. Adding a user to Kong in this way gives them access to
+Kong based on their group in the IdP.
+
+Starting with {{site.base_gateway}} version 2.7, admin accounts are created automatically 
```

Comment:
Do we want to say the version explicitly? I think we had agreed at some point to stop doing this (though we never documented it anywhere...)

Reply:
Removed